pkg/cloud/metadata/labels.go

```diff
@@ -53,15 +47,32 @@ const (
 	numWorkersPatchLabels = 10
 )
 
+// Initialized in ContinuousUpdateLabelsLeaderElection (depends on driver name).
+var (
+	// VolumesLabel is the label name for the number of volumes on a node.
+	VolumesLabel string
+
+	// ENIsLabel is the label name for the number of ENIs on a node.
+	ENIsLabel string
+)
+
 type enisVolumes struct {
 	ENIs    int
 	Volumes int
 }
 
+// initVariables initializes variables that depend on driver name.
+// Separated into a spearate function from ContinuousUpdateLabelsLeaderElection so it can be called in tests.
+func initVariables() {
+	VolumesLabel = util.GetDriverName() + "/non-csi-ebs-volumes-count"
+	ENIsLabel = util.GetDriverName() + "/enis-count"
+}
```

Comment:
If somebody has a way around this, I'm all ears, but my best idea is a significant refactoring of creating a standardized interface for all these names, which I opted not to do in this PR as it would massively increase the lines of code and scope.